opsdroid/message.py

```diff
@@ -11,10 +11,52 @@
 
 
 # pylint: disable=C0103
-def Message(text, user, room, connector, raw_event=None):
-    """
-    Stand-in function for the deprecated opsdroid.message.Message object.
+class Message(NewMessage):
+    """A message object.
 
-    Returns a new opsdroid.events.Message object using the same arguments.
+    Deprecated. Use ``opsdroid.events.Message`` instead.
     """
-    return NewMessage(user, room, connector, text, raw_event)
+
+    def __init__(self, text, user, room, connector,
+                 raw_message=None):  # noqa: D401
+        """Deprecated opsdroid.message.Message object."""
+        super().__init__(text, user, room, connector,
+                         raw_event=raw_message)
+
+    @property
+    def room(self):  # noqa: D401
+        """The room the message was sent into."""
+        warn(
```

Comment:
Aboved I asked about warnings vs _Logger, if we choose warnings, perhaps we should choose to either use `warninings.warn` or just `warn` throughout the codebase, what do you think?

Reply:
We should probably be using proper warnings (`warninings.warn`).